react-main/src/redux/login/actions.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import loginService from "../../services/loginService";
+import { history } from '../store';
+
+export const actions = {
+  GET_TOKEN_SUCCESS: 'GET_TOKEN_SUCCESS',
+  GET_TOKEN_FAILURE: 'GET_TOKEN_FAILURE'
+};
+
+const actionCreators = {
+  login: (credentials) => async dispatch => {
+    //dispatch({ type: actions.GET_BOOKS }); PREGUNTAR
+    const response = await loginService.login(credentials);
+    if (response.ok) {
+      dispatch ({
+        type: actions.GET_TOKEN_SUCCESS,
+        payload: response.data //esto trae un token de la api
```

Comment:
no need for this comment, but if you do need a comment somewhere please comment in english